Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1

```diff
@@ -1791,7 +1790,7 @@ https://psappdeploytoolkit.com
         New-ZipFile -DestinationArchiveDirectoryPath $configToolkitLogDir -DestinationArchiveFileName $DestinationArchiveFileName -SourceDirectory $logTempFolder -RemoveSourceAfterArchiving
     }
 
-    If ($script:notifyIcon) {
+    If (Test-Path -LiteralPath 'variable:notifyIcon') {
```

Comment:
What's the purpose behind this change?

Reply:
Same here, it's an attempt to access a variable that doesn't always exist depending on whether the toolkit is running silent or not. Under PowerShell's loosey-goosey default mode, it slips through but in strict compliance mode (`Set-StrictMode -Version Latest`), it causes a throw under the right conditions.